spec/models/spree/adjustment_spec.rb

```diff
@@ -61,40 +61,43 @@ module Spree
       describe "Shipment adjustments" do
         let(:shipping_method) { create(:shipping_method_with, :flat_rate) }
         let(:shipment)        { create(:shipment_with, :shipping_method, shipping_method: shipping_method) }
-        let!(:order)          { create(:order, distributor: hub) }
+        let(:order)           { create(:order, distributor: hub) }
         let(:hub)             { create(:distributor_enterprise, charges_sales_tax: true) }
-        let!(:line_item)      { create(:line_item, order: order) }
+        let(:line_item)       { create(:line_item, order: order) }
         let(:adjustment)      { order.adjustments(:reload).shipping.first }
 
-        before { order.shipments = [shipment] }
+        describe "the shipping charge" do
+          it "is the adjustment amount" do
+            order.shipments = [shipment]
 
-        it "has a shipping charge of $50" do
-          adjustment.amount.should == 50
+            adjustment.amount.should == 50
+          end
         end
 
         describe "when tax on shipping is disabled" do
+          before { Config.shipment_inc_vat = false }
```

Comment:
This is not reverted after the test, is it? Depending on the test execution order it could have confusing side effects. But that's not the scope of your pull request. So no need to change.